index.html

```diff
@@ -78,11 +56,11 @@ <h2 class="z-depth-3">What is Happening in Washington DC</h2>
 
 
   <!--Import jQuery before materialize.js-->
-  <script async defer
```

Comment:
Did you get the map to work after removing this? I think we need this in order for the Google Maps API to work

Reply:
I did not. I will put this back in. Thanks!